### PR TITLE
AC-6811: Impact-api has tests that use a UserRole to give staff permissions

### PR DESCRIPTION
**Pull request by shakirandagire**
**Changes introduced in AC-6811**
Make the tests pass that are affiliated to the updated to give staff permissions with clearances

**How to test**

**On development:**
Run the tests using the `make test` and notice that the 3 tests fail.

**Checkout to this branch:**
Run the tests using the `make test` and notice that the 3 tests pass.

### CODE REVIEW

**Comment by VincentHokie** [comment]
taking code

File: web/impact/impact/tests/api_test_case.py
Comment (VincentHokie) [inline web/impact/impact/tests/api_test_case.py]: the indentation looks off here..

File: web/impact/impact/tests/test_algolia_api_key_view.py
Comment (VincentHokie) [inline web/impact/impact/tests/test_algolia_api_key_view.py]: I don't think we need this anymore

File: web/impact/impact/tests/test_algolia_api_key_view.py
Comment (VincentHokie) [inline web/impact/impact/tests/test_algolia_api_key_view.py]: same with [this](https://github.com/masschallenge/impact-api/pull/301/files#r295203030)

File: web/impact/impact/tests/api_test_case.py
Comment (shakirandagire) [reply web/impact/impact/tests/api_test_case.py]: Let me rectify this

File: web/impact/impact/tests/test_algolia_api_key_view.py
Comment (shakirandagire) [reply web/impact/impact/tests/test_algolia_api_key_view.py]: Am on it

File: web/impact/impact/tests/test_algolia_api_key_view.py
Comment (shakirandagire) [reply web/impact/impact/tests/test_algolia_api_key_view.py]: Am on it

**Review by VincentHokie — APPROVED** [review]
✅

File: web/impact/impact/tests/test_algolia_api_key_view.py
Comment (jonkiparsky) [reply web/impact/impact/tests/test_algolia_api_key_view.py]: @VincentHokie @shakirandagire Just to be clear on this, we are maintaining the Staff ProgramRoleGrant for the time being, as it is important for (among other things) seeing navigation on the accelerate side. 
I'm not aware of any place where we use this on the impact side, so the question is whether we care more about symmetry of "staff" status across the two repos, or simplicity on the impact side. 
I'm okay with these changes as-is, I just wanted to make sure we were all on the same page about this.

**Review by jonkiparsky — APPROVED** [review]

File: web/impact/impact/tests/test_algolia_api_key_view.py
Comment (VincentHokie) [reply web/impact/impact/tests/test_algolia_api_key_view.py]: yup, we're on the same page @jonkiparsky . thanks for the context..

Comment (shakirandagire) [reply web/impact/impact/tests/test_algolia_api_key_view.py]: @jonkiparsky I think the symmetry of staff status across the two repos would be ideal. And as @VincentHokie has stated, we are on the same page.

Comment (jonkiparsky) [reply web/impact/impact/tests/test_algolia_api_key_view.py]: Great, thanks. So if you want to restore the staff Role Grants to maintain symmetry, I can hold off on merging until that's done, or I can merge now. Let me know which you'd prefer.

**Pull request merged**
